@vates/fatfs/structs.js

```diff
@@ -7,7 +7,7 @@ var _ = require('struct-fu'),
 var bootBase = _.struct([
     _.byte('jmpBoot', 3),
     _.char('OEMName', 8),
-    _.uint16le('BytsPerSec'),
+    _.uint16le('BytesPerSec'),
```

Comment:
It seems pretty clear that this typo comes from some underlying thing, but I'm not certain that retaining the typo is necessary.
https://www.vlsi.fi/player_vs1011_1002_1003/modularplayer/filesys_8c.html

I'm assuming that this is parsing a binary object and giving items field names.